components/UserDropdownMenu/UserDropdownMenu.jsx

```diff
@@ -0,0 +1,64 @@
+'use strict'
+
+require('./UserDropdownMenu.scss')
+
+const React = require('react')
+const Dropdown = require('../Dropdown/Dropdown.jsx')
+const userDropdownLists = [
+  [
+    { label: 'My Profile', link: 'javascript:;', id: 0 },
+    { label: 'Dashboard', link: 'javascript:;', id: 1 },
+    { label: 'Settings', link: 'javascript:;', id: 2 },
+    { label: 'Payments', link: 'javascript:;', id: 3 }
+  ],
+  [
+    { label: 'Help', link: 'javascript:;', id: 0 }
+  ],
+  [
+    { label: 'Log out', link: 'javascript:;', id: 0 }
+  ]
+]
+
+const UserDropdownMenu = {
```

Comment:
ES6 class for stateful components, see https://github.com/appirio-tech/react-components/commit/3118fed8f68e8ca4a511142d2cbdf982f9344a6d